UPGRADING.md

```diff
@@ -1,6 +1,11 @@
 Upgrading Grape
 ===============
 
+### Grape::Middleware::Base
```

Comment:
For later, we should have an `Upgrading >= X.Y.Z` here whatever the next version is.